src/data/data.cu

```diff
@@ -130,12 +130,12 @@ void MetaInfo::SetInfoFromCUDA(Context const&, StringView key, Json array) {
   }
   // uint info
   if (key == "group") {
-    auto array_interface{ArrayInterface<1>(array)};
+    ArrayInterface<1> array_interface{array};
```

Comment:
Curious why `auto` needed to be changed or is this just to be more explicit?

Reply:
Workaround for compiler error.